charts/osm/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.0
+version: 0.3.0
```

Comment:
For now, the chart version and the app version should be the same. Could you please update this to 0.4.0?

Reply:
Done, was hesitant to double increment it without knowing consequences.